.github/workflows/validate_json_schema.yaml

```diff
@@ -10,5 +10,5 @@ jobs:
       - name: Validate CTS against JSON Schema
         uses: cardinalby/schema-validator-action@v2
         with:
-          file: 'cts.json'
+          file: 'tests/**/*.json|cts.json'
```

Comment:
The fact that the individual JSON files are test suites in their own right, although a clever feature, somewhat cuts against the requirement that all implementers should consume the same set of tests.

Having a less sophisticated way of combining the individual JSON files into cts.json could avoid having to install node. It might be possible to use just a shell script or, failing that, a shell script that depends on just jq.

OTOH I can see that having the individual files conform to the same schema is a bonus for early detection of syntax errors.

Reply:
The implementers should consume the cts.json and nothing else.

JavaScript seemed the most natural language to operate with JSON. Operating with pieces of JSON as strings would be error prone IMO. So we will have to use some tool for that. Using an actual programming language (instead of a query language) seems beneficial should we decide to add more logic (e.g. version generation) to the build script in the future. 

Early detection is the main reason why I left the structure the same.

Reply:
> The implementers should consume the cts.json and nothing else.

Let's not ignore JSONPath implementors who also need to update the CTS.

Reply:
> Let's not ignore JSONPath implementors who also need to update the CTS.

That's what a readme or a CONTRIBUTING file is for.  I doubt implementors will have a hard time with it.